src/graphql/batches.js

```diff
@@ -48,10 +53,7 @@ query GET_ALL_BATCHES ($id: Int, $limit: Int, $start: Int, $sort: String, $state
 export const GET_BATCH = `
 query GET_BATCH ($id:ID!) {
   batch(id: $id) {
-    id
-    name
-    end_date
-    start_date
+    ${batchesFields}
     created_at
```

Comment:
shouldn't these also go into `batchesField`? the rest of the attributes that are listed here.